frontend/lib/DataModels/member_info.dart

```diff
@@ -0,0 +1,16 @@
+import 'User/user.dart';
+
+class MemberInfo {
```

Comment:
zastanawiam się nad jednym - czy w przypadku tak ściśle związanych modeli z adminem, nie powinniśmy zmienić lekko struktury katalogów, ale to na następny task. Część modeli jest rożrzucona w models, część w samych klasach